src/expr/src/vector_op/regexp.rs

```diff
@@ -406,12 +416,12 @@ fn regexp_replace(
                 }
             } else {
                 // `N` is not specified
-                if ctx.regex.captures(&text[start..]).is_none() {
+                if ctx.regex.captures(&text[start..]).is_err() {
```

Comment:
Shouldn't it be like this?
```suggestion
                if let Ok(None) = ctx.regex.captures(&text[start..]) {
```